docs/tutorials/cargo_tests/experimental/src/main.rs

```diff
@@ -0,0 +1,19 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! This is a demo project using test data with experimental features.
+//! 
+//! For more information, see the tutorial [cargo.md](../../cargo.md).
+
+use icu::segmenter::GraphemeClusterSegmenter;
+
+fn main() {
+    let segmenter = GraphemeClusterSegmenter::try_new_unstable(
+        &icu_testdata::unstable(),
+    )
+    .expect("Grapheme break data should be present in testdata");
```

Comment:
thought: it's unfortunate that the key being present in the data doesn't mean anything for `und`-only data